libs/domain/site/src/services/locale/default-locale.service.spec.ts

```diff
@@ -70,5 +70,22 @@ describe('DefaultLocaleService', () => {
         Intl.DateTimeFormat('de-DE').format(date)
       );
     });
+
+    describe('and showTime is true', () => {
```

Comment:
Thanks, @aleespryker 🥇 for adding unit tests for defects we report! It helps a lot in terms of regression and quality.
I hope everyone will do the same